process_pr.py

```diff
@@ -1476,11 +1534,11 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
 
     # end of parsing comments section
     cache_comment = None
-    if technical_comment:
-        cache_comment = technical_comment
+    if technical_comments:
+        cache_comment = technical_comments
     else:
         if already_seen:
-            cache_comment = already_seen
+            cache_comment = {0: already_seen}
```

Comment:
@iarspider , once `technical_comments` is changed to `list` then please change this to `[already_seen]`